plutus-tx-plugin/app/GeneratePluginOptionsDoc.hs

```diff
@@ -0,0 +1,44 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes       #-}
+
+module Main (main) where
+
+import Control.Lens
+import Data.Map qualified as Map
+import Data.Text (Text)
+import Data.Text qualified as Text
+import Data.Text.IO qualified as Text
+import PlutusTx.Options qualified as O
+import Prettyprinter
+import PyF (fmt)
+
+main :: IO ()
+main = Text.writeFile "doc/plutus/tutorials/plugin-options" optionsTable
```

Comment:
Maybe take the file path to write to as an arg, or just print to stdout? relying on the CWD being right can be a bit fragile.